commands/button.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
     name: 'button',
     description: 'Either gives you 50 gbp or -1500 gbp on use',
-    execute(message, args, master, buttonJSON){
-
+    execute(message, args, master, buttonJSON, command_stats){
+        console.log(command_stats)
```

Comment:
At some point it would be good to move away from console logs and use the node debugger but that can be done latter and swapped out in bulk.